torch/quantization/QConfig.py

```diff
@@ -2,19 +2,60 @@
 from collections import namedtuple
 from .observer import *
 from .fake_quantize import *
+import torch.nn as nn
 
-QConfig = namedtuple('QConfig',
-                     ['activation', 'weight'])
+class QConfig(namedtuple('QConfig', ['activation', 'weight'])):
+    """
+    Describes how to quantize a layer or a part of the network by providing
+    settings (observer classes) for activations and weights respectively.
 
-default_qconfig = QConfig(activation=default_observer(),
-                          weight=default_weight_observer())
+    Note that QConfig needs to contain observer **classes** (like MinMaxObserver), not the concrete instances.
```

Comment:
I think it's either **classes** or **functools.partial object**?

Reply:
or we can say, it's something that will return an instance when called. sounds a bit confusing though.